src/dex/services/JsonRpcService/utils.ts

```diff
@@ -11,6 +11,22 @@ export const solidityAddressToTokenIdString = (address: string): string => {
   }
 };
 
+export const solidityAddressToAccountIdString = (address: string): string => {
```

Comment:
@anshulkhare-lab49 Moves these functions to the `shared/utils `folder. We need to completely move away from having DAO and DEX cross dependencies (don't import from `dex` in `dao` and vice-versa).

Reply:
Moved the functions to shared/utils.